contents/docs/experiments/data-warehouse.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Data Warehouse
+---
+
+> 🚧 **Note:** Data warehouse integration is currently considered in `beta`. We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [daniel.b@posthog.com](mailto:daniel.@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
+
+Experiment metrics are always event-based: something happens at a certain point in time. If one of your data warehouse tables includes event-like data, you can use it as a primary or secondary metric for your trends experiment.
+
+To use a data warehouse table with an experiment, you'll first need to join the 'events' table to your data warehouse table:
+
+1. Navigate to Data warehouse and click on 'Add join' from the triple dot menu next to your table.
+2. Join the 'events' table to your data warehouse table:
+	a. Under 'Source Table Key', specify a column that holds the value of the `distinct_id` present for the `$feature_flag_called` event.
+	b. Check 'Optimize for Experiments' to ensure only the most recent matching event is joined to your table.
+	c. Under 'Source Timestamp Key', specify a column that represents the timestamp of the table row. It will be compared with the event timestamp to determine the most recent `$feature_flag_called` event for the row.
```

Comment:
```suggestion
1. Navigate to the [Data warehouse](https://us.posthog.com/data-warehouse) tab and click on **Add join** from the triple dot menu next to your table.
2. Join the `events` table to your data warehouse table:
    a. Under **Source Table Key**, specify a column that holds the value of the `distinct_id` present for the `$feature_flag_called` event.
    b. Check **Optimize for Experiments** to ensure only the most recent matching event is joined to your table.
    c. Under **Source Timestamp Key**, specify a column that represents the timestamp of the table row. It will be compared with the event timestamp to determine the most recent `$feature_flag_called` event for the row.
```

Reply:
Fixing the styling :)
![CleanShot 2024-12-05 at 08 55 25@2x](https://github.com/user-attachments/assets/b6f02df7-b153-4966-867f-dc76ef646dcd)

Reply:
> Fixing the styling :)

Brilliant, thanks!